### ISSUE TITLE
Increase Dapr logging verbosity from app -> sidecar

### ISSUE THREAD

**Issue by lynn-orrell** [dapr/dapr#1720]
As it stands today, there is very little logging from the dapr sidecar once initialization is complete. I would like to propose that we increase the the logging for calls between the application and its local sidecar instance (specifically runtime errors that may be occurring such as ERR_STATE_STORE_NOT_CONFIGURED). Since this is a runtime issue that is not going to correct itself automatically, it should be surfaced in not only the caller's log but also in the sidecar log.

I would like to propose the following:

- All non-successful calls from application to local dapr sidecar be logged in the sidecar logs
- In the case of Invoke calls, a sidecar log message should only be produced if there is an issue in calling the local sidecar (e.g., the path is incorrectly formatted). A log message should not be produced if the target of the invocation (remote sidecar) fails, but instead this error message should be passed back to the caller.

**Comment by youngbupark** [comment]
@lynn-orrell 

Thanks for the feedbacks. When we worked on structured logging, we had considered log enrichment like you mentioned. However, the logs you mentioned can be duplicated with traces so that we did not improve it. But I agree the point that not all customer would enable trace and log is more beneficial during the debugging in production scenario.

Instead of additional duplicated logic, we may be able to leverage OT [stdout exporter](https://github.com/open-telemetry/opentelemetry-go/tree/master/exporters/trace/stdout) to export the trace to log.

cc/ @shalabhms
